Logger/LogFactory.cs

```diff
@@ -2,10 +2,36 @@
 {
     public class LogFactory
     {
-        public BaseLogger CreateLogger(string className)
+        private string _FilePath { get; set; } = null!; 
+
+
+        public void ConfigureFileLogger(string fileLogger) {
+            _FilePath = fileLogger; 
+        }
+
+        public string GetFileLoggerPath() {
+            return _FilePath; 
+        }
+
+        public BaseLogger CreateLogger(string creationClass, string className)
         {
+
+
+            if (className == "ConsoleLogger")
```

Comment:
```suggestion
            if (className == "ConsoleLogger")
```
You should avoid using hard coded strings like this, use `nameof` instaed.